wremnants/muon_selections.py

```diff
@@ -97,8 +97,8 @@ def select_z_candidate(df, mass_min=60, mass_max=120):
 
     return df
 
-def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None, era = "2016PostVFP"):
-    df = df.Define("goodTrigObjs", f"wrem::goodMuonTriggerCandidate(TrigObj_id,TrigObj_filterBits,\"{era}\")")
+def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None, era = "2016PostVFP"):≈ß
```

Comment:
Is this a typo? I see a  "~beta" after the colon

Reply:
That's the weirdest thing, that character is not there in my editor. Maybe something weird happened in the text formatting, let me check.

Reply:
That change only existed on the remote branch and appeared out of nowhere. 
I'm slightly baffled but anyway, I fixed it.

Reply:
Could be similar to what happens with thunderbird from a Mac OS system, sometimes white spaces or special characters appear formatted differently for those who receive the messages if they use a different operating system. 
Anyway, now that it is fixed I think this is ready for merge, but we can still wait for Suvankar for the last word